packages/account-v2/src/containers/POAForm/AddressDetailsForm.tsx

```diff
@@ -32,7 +31,7 @@ export const AddressDetailsForm = ({ resubmitting }: TAddressDetailsForm) => {
     } = useSettings();
     const { error: documentUploadError, isLoading: isDocumentUploading, upload } = useDocumentUpload();
     const invalidate = useInvalidateQuery();
-    const { isMobile } = useBreakpoint();
+    const { isMobile } = useDevice();
```

Comment:
switch to isDesktop

Reply:
`isDesktop` and `isMobile` will be handle in the next PR later 🙏